webpack-build/src/user.model.ts

```diff
@@ -1,11 +1,16 @@
 import { AutoMap } from '@nartc/automapper';
-import {Type, Expose} from 'class-transformer';
-import { Address } from './address.model';
-import { Profile } from './profile.model';
 
-export class User {
+import Contact from './contact.model';
+import Gender from './gender';
+
+
+export default class User {
   firstName!: string;
   lastName!: string;
-  profile!: Profile;
-  addresses!: Address[];
+  middleName!: string | null;
+  isActive!: boolean;
+  contact!: Contact | null;
+  age?: number;
+  // @AutoMap(() => Gender) // enum hasn't supported 
```

Comment:
For `enum`, it's just a `string` or `number` ultimately. You might want to just `AutoMap()` instead of passing in a `typeFn` `() => Gender`. I haven't tested but I think `plainToClass` will freak out.

Reply:
Yeah, that's try, thanks a lot.